link/freplace_test.go

```diff
@@ -0,0 +1,62 @@
+package link
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/internal"
+	"github.com/cilium/ebpf/internal/testutils"
+)
+
+func TestFreplace(t *testing.T) {
```

Comment:
Is it possible to freplace a function in the kernel? If yes it'd be great to have a test that covers this.

Reply:
It is [not supported](https://elixir.bootlin.com/linux/v5.14-rc3/source/kernel/bpf/verifier.c#L13126).